app/core/Engine/controllers/RemoteFeatureFlagController/utils.test.ts

```diff
@@ -0,0 +1,182 @@
+import { ControllerMessenger } from '@metamask/base-controller';
+import {
+  RemoteFeatureFlagController,
+  RemoteFeatureFlagControllerMessenger,
+} from '@metamask/remote-feature-flag-controller';
+import { createRemoteFeatureFlagController } from './utils';
+
+describe('RemoteFeatureFlagController utils', () => {
+  let messenger: RemoteFeatureFlagControllerMessenger;
+
+  beforeEach(() => {
+    messenger =
+      new ControllerMessenger() as unknown as RemoteFeatureFlagControllerMessenger;
+    jest.clearAllMocks();
+  });
+
+  describe('createRemoteFeatureFlagController', () => {
+    it('should create controller with correct config when enabled', () => {
```

Comment:
Can we, once and for all, agree that we have to name tests according to our guidelines and not according to whatever AI outputs?
```suggestion
    it('creates controller with correct config when enabled', () => {
```